sysutils/puppet-agent/src/opnsense/service/conf/actions.d/actions_puppetagent.conf

```
@@ -0,0 +1,23 @@
+[start]
+command:/usr/local/etc/rc.d/puppet start
+parameters:
+type:script
+message:starting puppet agent
+
+[stop]
+command:/usr/local/etc/rc.d/puppet onestop
+parameters:
+type:script
+message:stop puppet agent
+
+[restart]
+command:/usr/local/etc/rc.d/puppet restart
+parameters:
+type:script
+message:restart puppet agent
+
+[status]
+command:/usr/local/etc/rc.d/puppet onestatus || exit 0
```

```suggestion
command:/usr/local/etc/rc.d/puppet onestatus; exit 0
```

splitting hairs but more consistent with other services

---

Actually I think it was added on purpose, otherwise configctl returned an error in certain situations when running the `status` command.

---

It’s making the exit status unconditional. As I said either things work, but the semicolon is the one that the majority of status commands already use.